go.mod

```diff
@@ -1,6 +1,6 @@
 module go.strv.io/net
 
-go 1.20
+go 1.22
 
 require (
```

Comment:
Before we release a new version, an update of dependencies would be nice.

Reply:
Good point 👍